cli/tests/test_revset_output.rs

```diff
@@ -745,23 +742,23 @@ fn test_revset_committer_date_with_time_zone() {
 
     work_dir
         .run_jj([
-            "--config=debug.commit-timestamp=2023-01-25T11:30:00-05:00",
+            "--config=debug.commit-timestamp=\"2023-01-25T11:30:00[-05:00]\"",
```

Comment:
nit: convert to a raw string to not avoid the needless escaping

Reply:
Fair enough.